airbyte-webapp/src/pages/destination/DestinationSettingsPage/DestinationSettingsPage.tsx

```diff
@@ -19,10 +19,7 @@ import styles from "./DestinationSettings.module.scss";
 export const DestinationSettingsPage: React.FC = () => {
   const params = useParams() as { "*": StepsTypes | ""; id: string };
   const destination = useGetDestination(params.id);
-  const { connections } = useConnectionList();
-  const connectionsWithDestination = connections.filter(
-    ({ destination: { destinationId } }) => destinationId === destination.destinationId
-  );
+  const { connections: connectionsWithDestination } = useConnectionList({ destinationId: [destination.destinationId] });
```

Comment:
Weird to me that these are prop-drilled in for the Source Settings.

Should probably choose one pattern or the other, but that's outside this PR :)